### ISSUE TITLE
[Feature Request] Verify Backups

### ISSUE THREAD

**Issue by fillwithjoy1** [NeoApplications/Neo-Backup#814]
**Is your feature request related to a problem? Please describe it.**
ALWAYS, something is MESSED UP everytime I try to use this app. Sometimes the backup isn't "complete" and some files are missing or it didn't back up the base.apk to the app... I just want it to work reliably 😭, but its hard

**Describe the primary solution**
Neo Backup should verify the backup is successful and that all apps were backed up successfully.

**Describe alternative solutions, if any**
Maybe write files into a zip file so that it isn't tampered with and during transfer between devices.

**Screenshots**
If you're willing to add demonstrative screenshots, you're welcome to do so.

**System Information (please complete the following information):**

- Device: Pixel 6 Pro
- Android Version: 14
- ROM: Stock
- App Version: 8.3.4

**Comment by machiav3lli** [comment]
I think maybe some specific factors on your device are breaking the integrity of the backups. Maybe a storage clean up or so? Noone can really help, if you provide no specific info, more so when those are your own setup issues.

A zip function (like Swift's paradigm) is planned for future releases, but no ETA.

**Comment by fillwithjoy1** [comment]
> I think maybe some specific factors on your device are breaking the integrity of the backups. Maybe a storage clean up or so? Noone can really help, if you provide no specific info, more so when those are your own setup issues.
> 
> A zip function (like Swift's paradigm) is planned for future releases, but no ETA.

Yeah I just realised that not all my files were transferred successfully, so I redone it again (Synching) and then after redoing the restore, it's all fine again. I think I just need some time to relax and not panic over my missing backups
